setup.py

```diff
@@ -0,0 +1,40 @@
+from setuptools import setup
+
+with open('README.md', 'r') as fh:
+    long_description = fh.read()
+
+setup(
+    name='borealis-fireworks',
+    version='0.1.0',
+    packages=['borealis', 'borealis.util'],
+    url='https://github.com/CovertLab/borealis',
+    license='MIT',
+    author='Jerry Morrison',
+    author_email='j.erry.morrison@gmail.com',
+    description='Run FireWorks workflows in Google Cloud',
+    long_description=long_description,
+    long_description_content_type='text/markdown',
+    install_requires=[
+        'google-cloud-logging>=1.14.0',
+        'google-cloud-storage>=1.25.0',
+        'docker>=4.1.0',
+        'FireWorks>=1.9.5',
+        'requests>=2.22.0',
+        'ruamel.yaml>=0.16.7',
+        'subprocess32>=3.5.4',
+    ],
+    classifiers=[
+        'Development Status :: 3 - Alpha',
+        'Intended Audience :: Developers',
+        'Intended Audience :: Information Technology',
+        'Intended Audience :: Science/Research',
+        'Intended Audience :: System Administrators',
+        'License :: OSI Approved :: MIT License',
+        'Operating System :: OS Independent',
+        'Programming Language :: Python',
+    ],
+    # package_data=?,
+    # python_requires=?,
+    # entry_points=?,
```

Comment:
No scripts for Borealis?

Reply:
What's the tradeoff between creating a script and using `python -m borealis.gce`? I guess the former is easier to invoke but adds stuff to the command namespace. I'm unsure whether to rename `gce.py` to `workers.py` or `vms.py` or something.

Reply:
I forget the difference between scripts and entry points at the moment but one of them installs the command for the whole system (or in our case the pyenv), and does not need to be available in the path to be found.